tutorials/5min.md

```diff
@@ -84,4 +84,4 @@ var plot = new tauCharts.Plot({
 
 [Example jsFiddle](http://jsfiddle.net/taucharts/d6L9hppe/)
```

Comment:
This fiddle is not a facet chart, it is the same as the previous example fiddle.